test/WebJobs.Extensions.Tests/Extensions/DocumentDB/DocumentDBConfigurationTests.cs

```diff
@@ -86,6 +118,33 @@ public void CreateContext_UsesDefaultRetryValue()
             Assert.Equal(DocumentDBContext.DefaultMaxThrottleRetries, context.MaxThrottleRetries);
         }
 
+        [Theory]
+        [InlineData(typeof(IEnumerable<string>), true)]
+        [InlineData(typeof(IEnumerable<Document>), true)]
+        [InlineData(typeof(IEnumerable<JObject>), true)]
+        [InlineData(typeof(JArray), false)]
+        [InlineData(typeof(string), false)]
+        [InlineData(typeof(List<Document>), false)]
```

Comment:
I was thinking that too, but then realized that string implements IEnumerable<char>, which wouldn't make sense to support. Once I see one special-case, it makes me nervous that there are a bunch of others, too.

Plus, how would I construct the types? If someone wanted a `MyCustomEnumerable<Document>`, I'd need to do some reflection to create it before returning it. It felt like a lot of work and testing for not a lot of value.

Plus, we can always expand that later.